ui/server/src/main/scala/pl/touk/nussknacker/ui/process/processingtypedata/ProcessingTypeDataReload.scala

```diff
@@ -10,6 +10,12 @@ trait ProcessingTypeDataReload {
 
 }
 
+trait Initialization {
```

Comment:
would it be bad to have just ProcessingTypeDataReload

Reply:
@mproch WDYT? I personally don't like messing interfaces with some "technical quirks" because it causes a lot of problems in all places where u mocking it, using delegation and so on. I was thinking about moving this trait to some other package like utils, but won't force to that.

Reply:
don't have a strong opinion on this one... I agree it's not so good to mix technical quirks, but OTOH - ProcessingTypeDataReload is technical quirk as a whole :D